test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -662,7 +662,7 @@ private void MixMatch(string dataPath)
             var binaryTrainer = mlContext.BinaryClassification.Trainers.AveragedPerceptron("Label", "Features");
 
             // Append the OVA learner to the pipeline.
-            dynamicPipe = dynamicPipe.Append(new Ova(mlContext, binaryTrainer));
+            dynamicPipe = dynamicPipe.Append(mlContext.MulticlassClassification.Trainers.OneVersusAll(binaryTrainer));
```

Comment:
>mlContext [](start = 45, length = 9)

can you update [https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md](https://github.com/dotnet/machinelearning/blob/master/docs/code/MlNetCookBook.md) as well?

Reply:
Sure. I wish though that this was a bit less awkward to keep these up to date. Or that we could just link samples from the .md itself directly, somehow.